markdown-to-slack-blocks-tool/Dockerfile

```
@@ -0,0 +1,14 @@
+FROM node:lts
```

```suggestion
FROM node:18
```
Just in case new LTS causes break when it's switched to v20